core/domain/exp_domain.py

```diff
@@ -276,8 +278,13 @@ class ExplorationChange(change_domain.BaseChange):
     }, {
         'name': CMD_ADD_TRANSLATION,
         'required_attribute_names': [
-            'state_name', 'content_id', 'language_code', 'content_html',
-            'translation_html'],
+            'state_name', 'content_id', 'language_code',
+            'needs_update', 'translation_html', 'data_format'],
```

Comment:
The existing suggestion in the database will not pass this validation check, right?

Reply:
Oh right. Would it make sense to create a new commit command in that case to handle future updates to translation (for suggestions and exp editor edits) and deprecate this one?

Reply:
Went ahead and created a new command. PTAL